### PR TITLE
[Word] Add Get word count snippet

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: samples/word/25-paragraph/get-word-count.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        }
+
+        async function setup() {
+          await Word.run(async (context) => {
+            // Let's insert a couple of paragraphs to illustrate the point.
+            context.document.body.clear();
+            context.document.body.insertParagraph("Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.", "Start");
+            context.document.body.insertParagraph("To make your document look professionally produced, Word provides header, footer, cover page, and text box designs that complement each other. For example, you can add a matching cover page, header, and sidebar. Click Insert and then choose the elements you want from the different galleries.", "End");
+
+            await context.sync();
```
Comment (Rick-Kirkham) [inline samples/word/25-paragraph/get-word-count.yaml]: This setup method seems to work even if this context.sync is commented out. Is it needed for some reason that I don't see?

File: samples/word/25-paragraph/get-word-count.yaml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+            context.document.body.paragraphs.getFirst().alignment = "Left";
```
Comment (Rick-Kirkham) [inline samples/word/25-paragraph/get-word-count.yaml]: What are these two lines for? I think paragraphs in Word are left-aligned by default.

File: samples/word/25-paragraph/get-word-count.yaml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                paragraph.split(" ").forEach((term) => {
+                  let currentTerm = term.trim();
+                  if (currentTerm) {
+                    text.push(currentTerm);
+                  }
+                });
+              }
+            });
+
+            let makeTextDistinct = new Set(text);
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply samples/word/25-paragraph/get-word-count.yaml]: @Rick-Kirkham: Reference - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

File: samples/word/25-paragraph/get-word-count.yaml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/25-paragraph/get-word-count.yaml]: I grabbed the setup content from another Word snippet. Depending on what we decided here, I'll clean those up too.

**Pull request merged**